_posts/2022-11-23-trino-summit-2022-apple-recap.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Trino at Apple"
+author: "Vinitha Gankidi, Yathi Peddyshetty, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/apple.jpg
+---
+
+This post continues [a larger series of posts]({% post_url 
+2022-11-21-trino-summit-2022-recap %}) on the Trino Summit 2022 sessions.
+Following the [Keynote: State of Trino session]({% post_url
+2022-11-22-trino-summit-2022-state-of-trino-keynote-recap %}), engineers from Apple shared the
+current usage of Trino at Apple. They discuss how they support Trino as a
+service for multiple end-users, and the critical features that drew Apple to
+Trino. They wrap up with some challenges they have faced and some development
+they have planned to contribute to Trino.
+
+<!--more-->
+
+{% youtube 3afcRK6Yvio %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Apple.pdf">
+  Check out the slides here!
+</a>
+
+## Recapping
+
+There's tremendous adoption across multiple teams using Trino at Apple. Use
+cases range from the typical adhoc analytics and BI analytics, to running A/B
+testing and most exciting is federated querying. 
+
+To deploy Trino as a service, Apple has an in-house k8s operator to manage the
+Trino cluster lifecycles. They also created an orchestrator to provision and
+simplify cluster creation and management. They make this a self-service console
+that allows user to provision their own clusters per request through a web UI in
+the orchestrator. It also takes care of autoscaling and other technical
+complexities in the system.
+
+Apple primarily uses Iceberg, Hive, and Cassandra connectors. They have a heavy
+focus on Apache Iceberg as their table format and have contributed a significant
+amount of PRs to improve interoperability between Trino and Spark and increased
+coverage of Iceberg APIs. Other challenges Apple face stem from lack of flexible
+routing of queries to achieve zero downtime, and having pluggable optimizer
+rules and operators.
+
+Apple has various features on their roadmap to eventually contribute to the
+community. This includes, implementing missing Iceberg APIs, support all 
+partition transforms predicate pushdowns, bucketed joins, simple aggregate
```

Comment:
what is a "partition transforms predicate pushdowns" ... are you just missing a comma maybe?

Reply:
missed a comma